src/components/networked.js

```diff
@@ -269,6 +280,20 @@ AFRAME.registerComponent('networked', {
     if (entityData[0] == 1) {
       entityData = Compressor.decompressSyncData(entityData, this.getAllSyncedComponents());
     }
+
+    if (entityData.lastOwnerTime < this.lastOwnerTime) {
+      return;
+    }
+
+    if (this.data.owner !== entityData.owner) {
+      // TODO: File issue for partial set attribute.
+      // this.el.setAttribute("networked", { owner: entityData.owner });
```

Comment:
Hmm, really strange that this didn't work, definitely worth investigating. Did the 3 argument version also not work? `this.el.setAttribute("networked", "owner", entityData.owner)`

Reply:
Yeah the three argument one failed as well. I'll file a bug in AFrame.

Reply:
I think I fixed this in #69

Reply:
Yup, that seems to have fixed it! Nice catch!